presto-docs/src/main/sphinx/admin/spill.rst

```diff
@@ -65,6 +65,27 @@ Presto treats spill paths as independent disks (see `JBOD
 <https://en.wikipedia.org/wiki/Non-RAID_drive_architectures#JBOD>`_), so
 there is no need to use RAID for spill.
 
+Spill Compression
+-----------------
+
+When spill compression is enabled (``spill-compression-enabled`` property in
+:ref:`tuning-spilling`), spilled pages will be compressed if they're sufficiently
+compressible. Enabling this feature can reduce the amount of disk IO at the cost
```

Comment:
Perhaps, specify what type of compression is being used.

Reply:
I'll mention that the compression works the same way as exchange compression, but don't want to explicitly mention the codec name. It's LZ4 for now, but only because that's hard-coded in the `PagesSerdeFactory`. If someone were to add the ability to configure the compression codec there, my documentation would become incorrect.